README.md

```diff
@@ -1,227 +1,302 @@
 
-For attribution of specific content found on this Slack on public channels, we ask that you ask the originator of the content for permission. If you don't receive consent in a reasonable time, we ask that you credit the "Mac Admins Slack."
+Content posted in this fashion results in a no-warning expulsion of the offending account from The Mac Admins Slack. Trolling which does not meet this high bar is managed via the usual content review process.
 
-**Consequence**: Sharing content from private channels is discouraged without permission of the private channel. 
+## 6. Don't Spam
 
-## Read the Room
+Posting the same message to multiple channels is considered spamming and is not allowed.
 
-Once onboard, you will likely find yourself in a popular channel with many members who sound like we've figured it out. We haven't. While advice on one channel might read definitive, it is one member's lessons, interpretations and learnings from their Mac Admin experience. While there is a diverse set of experiences within this community, we continue to learn at every level of expertise. The daily practice of being a Mac Admin is more art than science.
+See also the additional [Vendor guidelines].
 
-This is a large community with many different humans populating hundreds of channels. Different channels have organically developed distinct personalities. Before posting in a channel with hundreds of members, we suggest you take the time to read the room. Specifically:
+## 7. Be real
 
-* Read the last couple of days of messages.
-* Examine the channel topic for helpful tips.
-* Click on the channel details and read the about section to see how many members are present, what messages have already been pinned, and what files have been shared.
-* Use Slack's search feature. Your question has problably already been discussed, or a least something similar. The results might give you the answer or help you frame your question better.
+It is forbidden to impersonate an individual, or pretend to represent a company.
 
-**Consequence**: Posting the same message to multiple channels is spamming. Don't spam. 
+As a member of a company, is is strictly not allowed to impersonate a customer or potential customer for the purposes of promotion. Violations of this nature will be treated very seriously and may have consequences for the accounts of other members of your company.
 
-## Direct Message Etiquette
+## 8. Direct Message Etiquette
 
-A direct message ("DM") is a private message to one or more other members. Sending a DM to another member you don't know might be jarring for the receiver. Before sending a DM to a member you've never contacted, consider the following:
+A direct message ("DM") is a private message to one or more other members. Administators have no access to these messages and have no ability to delete them, but the CoC still applies to this content, as does the [Content Review Process](#7-content-review-process).
```

Comment:
I think it is import to clarify that Administrators, through the Owner Account, can access DMs. Maybe something like this:

`Administators have no standing* access to these messages and have no ability to delete them, but the CoC still applies to this content, as does the [Content Review Process](#7-content-review-process). * See [15. Administrator Information Access](#15-administrator-information-access) for details on how the Slack Workspace Owner account can potentially be used to access DMs`